data/materials/carbon-allotropes.json

```diff
@@ -4,6 +4,16 @@
     "graphite",
     "graphene",
     "diamond",
-    "carbon nanotube"
+    "carbon nanotube",
+    "amorphous carbon",
+    "graphenylene",
+    "buckminsterfullerenes",
+    "carbon nanobuds",
+    "schwarzites",
+    "vitreous carbon",
+    "atomic and diatomic carbon",
+    "carbon nanofoam",
+    "lonsdaleite",
+    "linear acetylenic carbon"
```

Comment:
Would it make sense to have these (eg. "buckminsterfullerenes") as singular, to be consistent with the original (eg. "graphene", "diamond")?

Reply:
I believe buckminsterfullerenes is a catch all category for multiple forms of 'ball like' molecules, not just the original C60 form. Hence the plural.